zerver/openapi/zulip.yaml

```diff
@@ -9022,39 +9048,41 @@ paths:
                 allOf:
                   - $ref: "#/components/schemas/JsonSuccessBase"
                   - additionalProperties: false
+                    required:
+                      - result
+                      - msg
                     properties:
                       result: {}
                       msg: {}
                       ignored_parameters_unsupported: {}
                       presence_last_update_id:
                         type: integer
-                        nullable: true
```

Comment:
This was only nullable in the zephyr presence codepath, which was a bug actually. I included a prep commit here fixing the bug so that this can pass the CI. I assume it'll be merged as part of #29999 soon.